src/Compilers/CSharp/Portable/Symbols/Tuples/TupleTypeSymbol.cs

```diff
@@ -749,7 +749,7 @@ private ImmutableArray<FieldSymbol> CollectTupleElementFields()
                 var field = (FieldSymbol)member;
                 var index = field.TupleElementIndex;
 
-                if (index >= 0 && index == MatchesCanonicalElementName(field.Name) - 1)
+                if (field.IsDefaultTupleElement)
```

Comment:
Yes, thanks!

---
In reply to: [78257458](https://github.com/dotnet/roslyn/pull/13713#discussion_r78257458) [](ancestors = 78257458)